code/game/objects/structures/tables_racks.dm

```diff
@@ -858,7 +858,13 @@
 	if(!density) //Because broken racks -Agouri |TODO: SPRITE!|
 		return 1
 	if(istype(mover) && mover.checkpass(PASSTABLE))
-		return 1
+		return TRUE
+	if(istype(mover))
+		if(mover.checkpass(PASSTABLE))
+			return TRUE
```

Comment:
These two conditions are already checked on the lines just above.

Reply:
ty for noticing, missed that :)